keyboards/keyprez/rhino/readme.md

```diff
@@ -0,0 +1,17 @@
+# rhino
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [Christian Sandven](https://github.com/yourusername)
+* Hardware Supported: *Pro micro compatible boards*
+* Hardware Availability: *TBA*
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader to flash new firmware?